pom.xml

```diff
@@ -50,7 +50,7 @@
     <properties>
         <sirocco.version>1.0</sirocco.version>
 
-        <powsybl-ws-dependencies.version>2.12.0</powsybl-ws-dependencies.version>
+        <powsybl-ws-dependencies.version>2.13.0-SNAPSHOT</powsybl-ws-dependencies.version>
```

Comment:
We use only release version in pom.xml ...
Wait for PR : https://github.com/powsybl/powsybl-network-store/pull/430 to be merged and a new release of powsybl-network-store client to be created, and then override the negwork-store client version in the pom.xml ...

Reply:
yes!